# motion def:
motion_file: demo.csv
# IMU params:
imu: 1
# sample frequency of simulated GNSS/IMU data:
sample_frequency:
    imu: 100.0
    gps: 100.0
# topic name:
topic_name: /sim/sensor/imu
topic_name_gt: /pose/ground_truth
# output rosbag path:
output_path: /workspace/data/gnss_ins_sim/allan_variance_analysis
# output name:
output_name: data.bag
ground_truth_file: /workspace/assignments/06-imu-navigation/src/imu_integration/data/gt.txt
